Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM helioncf/ubuntu-core
+RUN apt-get update && apt-get install -y \
+  nodejs \
+  git-core curl zlib1g-dev build-essential libssl-dev libreadline-dev libyaml-dev libsqlite3-dev sqlite3 libxml2-dev libxslt1-dev libcurl4-openssl-dev python-software-properties libffi-dev \
+  software-properties-common
+
+RUN add-apt-repository --yes ppa:brightbox/ruby-ng && apt-get update
+
+RUN apt-get --assume-yes install \
+      ruby-switch \
+      ruby2.1 \
+      ruby2.1-dev \
+      ruby2.1-doc
+
+RUN gem2.1 install bundler
```

Comment:
Also don't need `gem2.1` here. `gem` will be the standard ruby version. Disregard @shsu with regards to `ruby-switch`, since this container has only one version of Ruby. It's only needed if you have more than one (the first one you install will be the default otherwise).